apps/graphql/src/apps/itinerary/helpers/Itineraries.js

```diff
@@ -27,6 +28,28 @@ export const differenceInMinutes = (
   );
 };
 
+export const mapToOpaqueID = (input: any) => {
```

Comment:
What we will do with this `any`?

Reply:
I don't know... got Flow errors all over the place when putting it as ` ItinerariesOneWaySearchParameters | ItinerariesReturnSearchParameters`

Reply:
I'll look on it but, can you pls write tests for that?

Reply:
Done!

Reply:
Maybe I overlooked something but why `mapToOpaqueID` is not part of the `parseParametersNew` like other parameters parsing? Then we can map origin/destination ids directly there and just reduce this function to something like .
```
ids.map(id =>
  fromGlobalId(id),
)
```

Reply:
@JoseDuSV for consistency, as in the `Bookings` query is also done on the resolver instead of the `dataLoader`. However I think we should just move both to the `dataLoaders` for simplicity, as you said

Reply:
@JoseDuSV Moved everything to dataLoaders for simplicity.

Reply:
Oh, I see. Booking case looks quite similar. But there is no other required parsing (for now because this whole part is currently mocked), so it quite makes sense to do it directly inside resolver instead of wrapping just `fromGlobalId` to another function inside booking dataloader.

But for search we already have the parser with solved input differences so it seems reasonable to me to make this mapping also part of it. If we don't need `mapToOpaqueID` to be reusable somewhere else like mentioned `fromGlobalId`, but according to specific input this doesn't seems to be the case.

Reply:
I was thinking about, and @JoseDuSV notes make sense to me.

Reply:
So it has been changed 👍 way simpler this way